src/wrapper.ts

```diff
@@ -44,4 +44,7 @@ LABKEY.SchemaKey = API.SchemaKey;
 LABKEY.Security = API.Security;
 LABKEY.Specimen = API.Specimen;
 LABKEY.Utils = API.Utils;
-LABKEY.Visualization = API.Visualization;
\ No newline at end of file
+LABKEY.Visualization = API.Visualization;
+
+// Let our presence be known in experimental deployments. See README.md.
+console.log('LABKEY is now running @labkey/api.');
```

Comment:
This gets logged only once, right?  Or it is once per page?

Reply:
I've scratched logging anything in favor of exposing package information. If it's there then the user knows things are working. See `LABKEY.__package__`.

![Screen Shot 2019-10-04 at 11 53 17 PM](https://user-images.githubusercontent.com/3926239/66251189-36643b00-e702-11e9-9fda-24c98972dfc4.png)

Updated README accordingly.

Reply:
Nice.  I like it.